tests/epics/adcore/test_writers.py

```diff
@@ -85,9 +85,19 @@ async def test_stats_describe_when_plugin_configured(
         hdf_writer_with_stats._plugins[0].nd_attributes_file,
         str("""<?xml version='1.0' encoding='utf-8'?>
 <Attributes>
-    <Attribute name="mydetector-sum" type="PARAM" source="TOTAL" addr="0"
-    datatype="DOUBLE" description="Sum of each detector frame" />
-</Attributes>"""),
+    <Attribute
+        name="mydetector-sum"
+        type="PARAM"
+        source="TOTAL" addr="0"
+        datatype="DOUBLE"
+        description="Sum of each detector frame" />
+    <Attribute
+        name="mydetector-Temperature"
+        type="EPICS_PV"
+        source="ca://LINKAM:TEMP"
```

Comment:
I don't think this should have the `ca://` prefix on it, unless areaDetector supports it? I don't think the tests had this before...

Reply:
It didn't have it before ..I'll remove it